e2e-nodejs/group-contracts/test-contracts-write-mint-a-pkp-and-set-scope-1-2-easy.mjs

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import { success, fail, testThis } from '../../tools/scripts/utils.mjs';
+import LITCONFIG from '../../lit.config.json' assert { type: 'json' };
+import { LitContracts } from '@lit-protocol/contracts-sdk';
+import { ethers } from 'ethers';
+import { AuthMethodType, AuthMethodScope } from '@lit-protocol/constants';
+
+export async function main() {
+  // ========== Controller Setup ===========
+  const provider = new ethers.providers.JsonRpcProvider(
+    LITCONFIG.CHRONICLE_RPC
+  );
+
+  const controllerWallet = new ethers.Wallet(
+    LITCONFIG.CONTROLLER_PRIVATE_KEY,
+    provider
+  );
+
+  // ==================== LitContracts Setup ====================
+  const contractClient = new LitContracts({
+    signer: controllerWallet,
+  });
+
+  await contractClient.connect();
+
+  // ==================== Test Logic ====================
+  const mintInfo = await contractClient.mint({
```

Comment:
Maybe we add a new namespace for higher level utility methods since we namepsace by contract name. We should call this method some other than mint as we are combining minting and permissions which is important for this function name to indicate